backend/index.js

```diff
@@ -9,15 +9,37 @@ const io = require("socket.io")(server, {
         methods: ["GET", "POST"]
     }
 });
-const port = 4001;
+const MafiaGame = require("./domain/MafiaGame");
+const Player = require("./domain/Player");
+
 
 
 // Listen for a "connection" event for incoming sockets.
 io.on("connection", (socket) => {
     console.log("User has connected");
+/*********************  Create Lobby and lobby code *******/
+socket.on("create-lobby", (nickname) => {
+    // create game and assign first player to the room
+    // only create new game if no game exists
+    let mafiaGame;
+    if (!mafiaGame) {
+         mafiaGame = new MafiaGame(); 
+    }
+    let roomID = mafiaGame.newGame();
+    let host = new Player(socket.id,roomID,nickname);
+    mafiaGame.gameRoomsDict[roomID].addPlayer(host);
```

Comment:
doesnt adding a player to a room create an event?

not sure in this case since host is first person.